subsys/net/l2/ethernet/lldp/lldp.c

```diff
@@ -117,10 +116,8 @@ static int lldp_send(struct ethernet_lldp *lldp)
 
 	net_pkt_frag_add(pkt, frag);
 
-	net_buf_add(frag, sizeof(struct net_lldpdu));
-
-	if (!net_pkt_write(pkt, frag, 0, &pos, sizeof(struct net_lldpdu),
-			   (u8_t *)lldp->lldpdu, BUF_ALLOC_TIMEOUT)) {
+	if (!net_pkt_append_all(pkt, sizeof(struct net_lldpdu),
```

Comment:
The net_pkt_append_all() is removed, so this needs to use net_pkt_write_new() instead.

Reply:
Updated